mlflow/pipelines/steps/split.py

```diff
@@ -61,10 +61,34 @@ def _get_split_df(input_df, hash_buckets, split_ratios):
     return train_df, validation_df, test_df
 
 
+def _parallelize(data, func, num_of_processes=8):
```

Comment:
Can we introduce a constant at the top of the file? e.g.
`_MULTI_PROCESS_POOL_SIZE = 8` for readability and patchability purpose?

Reply:
@jinzhang21  sorry had automerge on :/ -- will create a followup PR for this